README.md

```diff
@@ -1 +1,25 @@
-# mysqld_exporter (UNDER CONSTRUCTION)
+# mysqld_exporter
+
+Exporter for MySQL server metrics http://prometheus.io/
+
+## Building and running
+
+    make
+    ./mysqld_exporter <flags>
+
+## Configuration
+The configuration is in JSON. An example with all possible options:
```

Comment:
I agree, `config` is extraneous.

I think the configuration file is fine for now. It won't be a big deal to add other ways later if desired.